includes/block-patterns/lesson/templates/discussion-question.php

```diff
@@ -0,0 +1,69 @@
+<?php
+/**
+ * Discussion Question pattern content.
+ *
+ * @package sensei-lms
+ */
+
+if ( ! defined( 'ABSPATH' ) ) {
+	exit; // Exit if accessed directly.
+}
+
+?>
+<!-- wp:group {"backgroundColor":"tertiary"} -->
+<div class="wp-block-group has-tertiary-background-color has-background"><!-- wp:paragraph {"placeholder":"<?php esc_html__( 'Write your discussion question topic or prompt here.', 'sensei-lms' ); ?>"} -->
+<p></p>
+<!-- /wp:paragraph --></div>
+<!-- /wp:group -->
+
+<!-- wp:comments-query-loop -->
```

Comment:
Looking here, this block is only available on WP 6.0. But it's available for usage in older WP versions (like WP 5.9, for instance).

Are we going to have some way to avoid showing this pattern on older WP versions? Technically, we need to support even WP 5.8, so having some way to show/hide the pattern according to the WP version supported would be cool, here..

Reply:
Ah, good catch! cc @burtrw

Should we remove this pattern? Or maybe add a check before registering this pattern as @fjorgemota suggested?